docs/source/ruletypes.rst

```diff
@@ -629,6 +647,16 @@ This value will be used if ``generate_kibana_discover_url`` is true and ``kibana
 
 (Optional, string, default ``http://<es_host>:<es_port>/_plugin/kibana/``)
 
+opensearch_url
```

Comment:
Please keep the opensearch_* properties together, rather than interleaved among the kibana_* properties. This will be easier for people to read.

Reply:
fixed